lib/calculate_bill.rb

```diff
@@ -0,0 +1,42 @@
+class CalculateBill
+
+  def final_total(all_order)
+    over_all_cost(all_order).inject(:+)
+  end
+
+  def itemised_bill(all_order)
+    all_order.collect{|x| p itemised_cost(x)}
+  end
+
+  private
+
+  def match_item_to_price(order)
+   item = MENU.map{|k,v| v.to_f if order.keys.include? k}
+   item.compact.join.to_f
+  end
+ 
+  def type_of_pizza(order)
+    order.map{|k,v| k}.join
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)